spec/lib/qualifications_api/teacher_spec.rb

```diff
@@ -80,6 +80,38 @@
       )
     end
 
+    it "returns human readable ITT result values" do
+      {
+        "DeferredForSkillsTests" => "Deferred for skills tests",
+        "ApplicationReceived" => "Application received",
+        "ApplicationUnsuccessful" => "Application unsuccessful",
+        "InTraining" => "In training",
+        "NoResultSubmitted" => "No result submitted",
+        "UnderAssessment" => "Under assessment",
+      }.each do |k, v|
+        api_data["initialTeacherTraining"][0]["result"] = k
+        teacher = described_class.new(api_data)
+        qualifications = teacher.qualifications
+        expect(qualifications.find { |q| q.type == :itt }.details.result).to eq(v)
+      end
+    end
+
+    it "returns human readable Induction status values" do
+      {
+        "FailedInWales" => "Failed in Wales",
+        "InductionExtended" => "Induction extended",
+        "InProgress" => "In progress",
+        "NotYetCompleted" => "Not yet completed",
+        "PassedInWales" => "Passed in Wales",
```

Comment:
Should either of these `Wales` be `wales`, to test that particular `sub` call `CoercedDetails`?

Reply:
I've gone with the specific enum values given by the DQT API team here. 
`underscore.humanize` will convert `PassedInWales` to `Passed in wales` so the sub is acting on this value, though I agree it's not that obvious in the test. 
Could change one of these to make this more obvious?

Reply:
Ah, understood 👍🏼 . The test refactor you've added since makes this clear enough I think.